gcp/telemetry.ts

```diff
@@ -185,35 +214,10 @@ class Grafana {
           {
             provider: k8sProvider,
             customTimeouts: { create: "30m", update: "30m", delete: "30m" },
-            dependsOn: [cluster],
+            dependsOn: [cluster, ...dashboardsConfigMaps],
```

Comment:
In GKE, for some reason, it was taking longer to initialize the Grafana deployment because the ConfigMaps were taking a little too long to be created. So I'm reverting the dependency here.